db/migration/20210530121921_create_shares_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose"
+)
+
+func init() {
+	goose.AddMigration(upCreateSharesTable, downCreateSharesTable)
+}
+
+func upCreateSharesTable(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+create table share
+(
+	id             varchar(255) not null primary key,
+	jwt            varchar(255) not null unique,
```

Comment:
I gave some more thought to this and I think a JWT would be too long for a shareable link. I think a best approach would be to use something like [nano-id](https://github.com/matoous/go-nanoid). We can discuss that better when we are about to implement the `create share` functionality. For now, I think it is better to name this field as `name`

Reply:
Sounds good. Done.